nipype/utils/config.py

```diff
@@ -64,7 +67,6 @@
 parameterize_dirs = true
 poll_sleep_duration = 2
 xvfb_max_wait = 10
-profile_runtime = false
```

Comment:
shouldn't we add the resource_monitor options here?

Reply:
I have added a `default` option to `nipype.config.get` *ala* `dict.get`

Reply:
but shouldn't it also be here in the `default_cfg`?

Reply:
not necessarily